lte/gateway/c/core/oai/tasks/s11/s11_mme_bearer_manager.cpp

```diff
@@ -508,9 +507,8 @@ status_code_e s11_mme_create_bearer_response(
                  NwGtpv2c.h etc.. */
 
   for (int i = 0; i < response_p->bearer_contexts.num_bearer_context; i++) {
-    rc = gtpv2c_bearer_context_within_create_bearer_response_ie_set(
+    gtpv2c_bearer_context_within_create_bearer_response_ie_set(
```

Comment:
Instead of assert, we can add log

Reply:
I did not add the log here because this function call will either assert or return True always and I did not remove the DevAsserts everywhere because these are used everywhere in s11 and gtpv1-u. Removing them should be a separate effort and moreover this code is explicitly not owned by us and in current scope we have the capacity of only validate the compilation and unittests but end to end testing is out of scope.